src/site/content/en/blog/sanitizer/index.md

```diff
@@ -138,16 +120,16 @@ The following options specify how the sanitization result should treat the speci
 ```js
 const str = `hello <b><i>world</i></b>`
 
-new Sanitizer().sanitizeFor("div", str)
+$div.setHTML(new Sanitizer(), str)
```

Comment:
[Here and below:]

The syntax is:
.setHTML(string, {sanitizer: s});

For the specific example:
$div.setHTML(str, {sanitizer: new Sanitizer()});

However, the "new Sanitizer()" is the default. 

Maybe we can teach it this way:
- default case .setHTML(str);   // note: no sanitizer. Still sanitizes, but with default settings.
- other cases: .setHTML(str, {sanitizer: new Sanitizer({....})});  // advanced usage.